work/lessonbuilder_rewrite_urls.py

```diff
@@ -21,6 +21,17 @@
 from config.logging_config import *
 from lib.utils import *
 
+def fix_unwanted_url_chars(currenturl, url_prefix):
+    # parse url prefix, get path with https and path parsed_url.netloc + parsed_url.path
+    parsed_url = urlparse(url_prefix)
+    if url_prefix in currenturl:
```

Comment:
Keep this if check in the code below rather than here, otherwise you'd need to return the currenturl unchanged and the calling code would set rewrite = True unnecessarily. For the case where the links aren't to Vula but elsewhere.